presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunner.java

```diff
@@ -95,7 +98,28 @@ public ContainerQueryRunner(int coordinatorPort, String catalog, String schema,
         workers.forEach(GenericContainer::start);
 
         logger.info("Presto UI is accessible at http://localhost:" + coordinator.getMappedPort(coordinatorPort));
+
         TimeUnit.SECONDS.sleep(5);
+
+        String url = String.format("jdbc:presto://localhost:%s/%s/%s?%s",
+                coordinator.getMappedPort(coordinatorPort),
+                catalog,
+                schema,
+                "timeZoneId=UTC");
+
+        try {
+            Connection connection = DriverManager.getConnection(url, "test", null);
+            statement = connection.createStatement();
+        }
+        catch (Exception e) {
```

Comment:
What is thrown here?  If it's an IOException, can we let it simply throw?

Reply:
It is an SQLException.